backend/processing/bundle.bash

```diff
@@ -13,16 +13,17 @@ fi
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 work_dir="$(mktemp --directory)"
+asset_root='/asset-output'
```

Comment:
Good idea, but since this doesn't actually fix a bug I'd change the commit message to something like "refactor: Pull out variable"

Reply:
Good point about commit messages in general. I think I would squash all commits before merge to master anyway, so suggest to leave it as is for now.